requirements/requirements-dev.txt

```
@@ -1,9 +1,9 @@
-matplotlib~=3.4.3
+matplotlib~=3.5.0rc1
 numpy~=1.21.2
 fire~=0.4.0
 plotly~=5.3.1
 pytest~=6.2.5
-coverage~=5.5
+coverage~=6.0
```

There's already a dependabot PR for this, wait for it to merge. 
```suggestion
coverage~=5.5
```